archive.php

```diff
@@ -1,40 +1,43 @@
 <?php
+
 /**
  * The template for displaying Archive pages.
  *
  * Used to display archive-type pages if nothing more specific matches a query.
  * For example, puts together date-based pages if no date.php file exists.
  *
- * Learn more: http://codex.wordpress.org/Template_Hierarchy
- *
- * Methods for TimberHelper can be found in the /lib sub-directory
+ * Learn more: https://developer.wordpress.org/themes/basics/template-hierarchy/
  *
- * @package  WordPress
- * @subpackage  Timber
- * @since   Timber 0.2
  */
 
-$templates = array( 'archive.twig', 'index.twig' );
+namespace Timber\StarterTheme;
+
+use Timber\Timber;
+
+$templates = array('archive.twig', 'index.twig');
```

Comment:
Short array syntax everywhere?

Reply:
I would like to run Code standards after this one is merged directly at the whole codebase: https://github.com/timber/starter-theme/pull/159

Reply:
@Levdbas Makes sense 👍